packages/sdk/src/swap/services/ApiService.ts

```diff
@@ -59,14 +59,16 @@ const getQuote: BackendQuery<
   QuoteResponse
 > = async (baseUrl, quoteRequest, { signal }) => {
   const { brokerCommissionBps, ...returnedRequestData } = quoteRequest;
-  const params: QuoteQueryParams = {
+  const params = {
     amount: returnedRequestData.amount,
+    srcChain: returnedRequestData.srcChain,
     srcAsset: returnedRequestData.srcAsset,
+    destChain: returnedRequestData.destChain,
     destAsset: returnedRequestData.destAsset,
     ...(brokerCommissionBps && {
       brokerCommissionBps: String(brokerCommissionBps),
     }),
-  };
+  } as QuoteQueryParams;
```

Comment:
todo: revert this in https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/550